src/staff_augmentation/components/GetFreeQuote/GetFreeQuoteConfirmationMessage/index.js

```diff
@@ -0,0 +1,23 @@
+import SVGIcon from 'main_app/components/SVGIcon'
+
+import classes from './styles.module.scss'
+
+function GetFreeQuoteConfirmationMessage () {
+  return (
+    <>
+      <figure className={classes.confirmationBadge}>
+        <SVGIcon name='staff_augmentation/check_sign' className={classes.checkSign} />
+      </figure>
```

Comment:
This is not exactly a `figure`, because it's a decorative element. Figures are images whose content is important for the person viewing it, and they also have a caption. Change it to a `div`, which is meant for visual purposes.